__tests__/components/ResourceCreation/TestResourceCreation.test.tsx

```diff
@@ -172,6 +180,18 @@ describe('TestResourceCreation', () => {
     const procedureResource = screen.getByText(/1. Procedure/i);
     expect(procedureResource).toBeInTheDocument();
 
+    const resourceCode = screen.getByText(/123: Colectomy/i);
+    expect(resourceCode).toBeInTheDocument();
+
+    await act(async () => {
+      fireEvent.mouseOver(resourceCode);
+    });
+
+    waitFor(() => {
+      const resourceCodes = screen.getAllByText(/123: Colectomy/i);
```

Comment:
Suggest using the `getByRole` function instead of `getAllByText` to make sure that something that is of the `tooltip` ARIA role comes up. This makes the testing act more like it is a user using the page.